<!DOCTYPE HTML>
<html xmlns:th="http://www.thymeleaf.org"
	xmlns:layout="http://www.ultraq.net.nz/web/thymeleaf/layout"
	layout:decorator="layout">
<body>
	<h1 layout:fragment="header">G2S (Genome to Structure)</h1>
	<div layout:fragment="content" class="container-fluid">
		<h2>
			<p> An auto-updating web-service to annotate genomic variants on
				protein structures</p>
		</h2>
		<p>
			There is a great interest and need of mapping and analyzing germline
			and somatic genomic variants generated by recent large-scale
			sequencing efforts in the context of 3D protein structures. Several
			tools have been developed to address the need, including SIFTS (<a
				href="https://www.ebi.ac.uk/pdbe/docs/sifts/">https://www.ebi.ac.uk/pdbe/docs/sifts/</a>)
			, a service by PDB (<a
				href="http://www.rcsb.org/pdb/chromosome.do">http://www.rcsb.org/pdb/chromosome.do</a>)
			, and G23D (<a href="http://www.sheba-cancer.org.il/G23D">http://www.sheba-cancer.org.il/G23D</a>).
			However, none of them provides a web service that support automated
			analysis programmatically. 
		</p>
		<p>
		    We have developed a comprehensive web service that support automated mapping and annotating genomic
			variants in 3D protein structures,
		</p>
		<p>
			There are two components in G2S web services:
		</p>
		<ul>
		     <li><b>G2S API</b> (<a th:href="@{/pageapi}" href="api.html">link</a>):       Query by Uniprot/Human Ensembl/Genomic Position. </li>
		     <li><b>Sequence Web Interface</b> (<a th:href="@{/sequence}" href="sequence.html">link</a>) : Query by any protein sequences. </li>
		</ul>
		<p>
		     G2S web service supports:
		</p>
		<ul>
			<li>Retrieving aligned protein structure chains for an
				Ensembl/UniProt entry.</li>
			<li>Retrieving aligned protein structure chains for any protein
				sequence.</li>
			<li>Retrieving residue mapping between protein sequences and
				protein structures.</li>
			<li>Retrieving structural position of a genomic variant.</li>
		</ul>
		<p>
			The web service will return high confidence pre-calculated mapping of
			the protein/residue aligned to the protein structures from the PDB in
			the JSON format. A fixed e-value score threshold are predefined with
			several alignment quality metrics to enable further filtering if
			desired. Our system makes frequent (weekly) updates to the set of
			pre-computed alignments as new pdb structures are added to RCSB,
			while keeping the set of protein sequences from public protein
			sequence databases fixed. Less frequently, the system is
			re-initialized using an updated set of protein sequences, and during
			re-initialization an all-against-all alignment process is required.
			This web service was initially designed and implemented for mapping
			and visualizing cancer mutations in the cBioPortal, a leading
			resource for studying cancer genomics (<a
				href="http://www.cbioportal.org/">http://www.cbioportal.org/</a>).
			The cBioPortal has been used by thousands of researchers (currently
			there are in average about 3,000 visit daily), mostly individuals
			outside of the authorsâ€™ group. Recently, we separated and generalized
			the web service from the cBioPortal and established the G2S web
			service in the project Genome Nexus for annotating genomic variants.
			The web service will continue serving the cBioPortal as well as any
			other applications. The codes of the web service and website are
			open-source and available at <a
				href="https://github.com/cBioPortal/pdb-annotation">GitHub</a>.
		</p>
		<p>
		
		<h4>
			<b>History:</b>
		</h4>
		</p>
		<li>2018-10-05 - Add support of ENST/ENSG mapping in all
			versions.</li>
		<li>2018-10-04 - Add support of mapping dbSNP ID to PDB.</li>
		<li>2018-06-30 - Fix small bug of mapping hgvs-grch37 genes.</li>
		<li>2017-11-30 - Initial Version released.</li>
		<p>
		
		<h4>
			<b>Citation:</b>
		</h4>
		</p>
		<p>
			G2S: A web-service for annotating genomic variants on 3D protein
			structures, Juexin Wang, Robert Sheridan, S Onur Sumer, Nikolaus
			Schultz, Dong Xu, Jianjiong Gao, Bioinformatics, 34(11), 1949-1950, <a
				href="https://doi.org/10.1093/bioinformatics/bty047">https://doi.org/10.1093/bioinformatics/bty047</a>
		</p>
	</div>
</body>
</html>